third_party/libadblockplus/grit/resources/elemhide_for_selector.jst

```diff
@@ -0,0 +1,9 @@
+// {{filename_with_query}} and {{url}} will be replaced with actual variable values
```

Comment:
I also thought to move that snippet to a function, perhaps even to place it into *elemhide.js* and here left just a single line which is calling that function and passes the values.

Reply:
there is no sense - one have to generate js and actually add variables as code body.